packages/web-components/src/calendar/calendar.template.ts

```diff
@@ -29,129 +30,148 @@ export function calendarRightPanelTitleTemplate<T extends Calendar>(): ViewTempl
+ * Interactive calendar secondary panel template using DataGrid
  * @param context - The templates context
  * @param todayMonth - a numeric representation for today's month
  * @param todayYear - a numeric representation for today's year
- * @returns - interactive calendar right panel template for month or year picker
+ * @returns - interactive calendar secondary panel template for month or year picker
  *
  * @internal
  */
-export function interactiveRightPanelGridTemplate<T extends Calendar>(
+export function interactiveSecondaryPanelGridTemplate<T extends Calendar>(
   options: CalendarOptions,
   todayMonth: number,
   todayYear: number,
 ): ViewTemplate<T> {
   const gridTag = html.partial(tagFor(options.dataGrid));
 
   return html<T>`
-  <${gridTag} class="right-panel-grid interact" part="right-panel-grid" generate-header="none">
+  <${gridTag} class="secondary-panel-grid interact" part="secondary-panel-grid" generate-header="none">
       ${x =>
         x.yearPickerOpen
           ? html`${repeat(
               x => x.getDecadeText(x.getYearPickerInfo().decadeStart),
-              rightPanelRowTemplate(options, todayMonth, todayYear),
+              secondaryPanelRowTemplate(options, todayMonth, todayYear),
             )}`
-          : html`${repeat(x => x.getMonthText(), rightPanelRowTemplate(options, todayMonth, todayYear))}`}
+          : html`${repeat(x => x.getMonthText(), secondaryPanelRowTemplate(options, todayMonth, todayYear))}`}
   </${gridTag}>
   `;
 }
 
 /**
- * The template for the right panel.
+ * The template for the secondary panel.
  *
  * @param context - Element definition context for getting the cell tag for calendar-cell
- * @returns - a template for the right panel
+ * @returns - a template for the secondary panel
  * @public
  */
-export function rightPanelTemplate<T extends Calendar>(options: CalendarOptions): ElementViewTemplate<T> {
+export function secondaryPanelTemplate<T extends Calendar>(options: CalendarOptions): ElementViewTemplate<T> {
   const today: Date = new Date();
   const todayMonth: number = today.getMonth() + 1;
   const todayYear: number = today.getFullYear();
-  return html<T>` ${interactiveRightPanelGridTemplate(options, todayMonth, todayYear)} `;
+  return html<T>` ${interactiveSecondaryPanelGridTemplate(options, todayMonth, todayYear)} `;
 }
 
 /**
  * The template for the Calendar component.
  * @public
  */
 export const template: ElementViewTemplate<Calendar> = html`
-  <div class="control">
+  <div class="control" @keydown=${(x, c) => x.handleControlKeydown(c.event as KeyboardEvent)}>
     <div class="date-view">
       <div class="header">
         ${calendarTitleTemplate()}
         <div class="navicon-container">
           <span
             class="navicon-up"
             part="navicon-up"
-            @click="${(x, c) => x.switchMonth(x.getMonthInfo().previous.month, x.getMonthInfo().previous.year)}"
+            tabindex="1"
```

Comment:
How do this function without a tabindex or with tabindex set to '0'?

Reply:
I changed their tabindices to 0 and it still works :) Thanks for the tip!